test/fuzz/SavingsTest.t.sol

```diff
@@ -283,7 +284,9 @@ contract SavingsTest is Fixture, FunctionUtils {
         uint256 indexReceiver,
         uint256[2] memory elapseTimestamps
     ) public {
-        for (uint256 i; i < amounts.length; i++) amounts[i] = bound(amounts[i], 0, _maxAmount);
+        for (uint256 i; i < amounts.length; i++) {
```

Comment:
IS it the linter that adds those brackets?

Reply:
It is me, revert back to without brackets